src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -157,7 +159,7 @@ public String toString(@Nullable Event event, boolean debug) {
 			default:
 				assert false;
 		}
-		return "the last " + word + " " + type;
+		return "last " + word + " " + type;
```

Comment:
```suggestion
		return "the last " + word + " " + type;
```
it's better to be verbose for toString IMO